viper/plugin.spec.yaml

```diff
@@ -3,12 +3,12 @@ extension: plugin
 products: [insightconnect]
 name: viper
 title: "Viper"
-description: "The Viper plugins allows you to use Viper's platform to store, search, and analyze files"
+description: Utilize Viper's platform to store, search, and analyze files
```

Comment:
Make sure this plugin builds with the apostrophe. I'm inclined to say this is now broken.